### ISSUE TITLE
Double class selector doesn't work?

### ISSUE THREAD

**Issue by xavi** [seek-oss/treat#123]
I'm using the code below to do a CSS selector like `.checked.readonly` but it doesn't work...

```
export const readonly = style({
  pointerEvents: 'none'
})
export const checked = style(theme => ({
  backgroundColor: 'white',
  selectors: {
    [`${readonly}&`]: {
      backgroundColor: 'red'
    }
  }
}))
```

Is there a bug in Treat or the code is not correct?

**Comment by maraisr** [comment]
I think you have to apply the `&` first;

```diff
export const readonly = style({
  pointerEvents: 'none'
})
export const checked = style(theme => ({
  backgroundColor: 'white',
  selectors: {
-    [`${readonly}&`]: {
+    [`&${readonly}`]: {
      backgroundColor: 'red'
    }
  }
}))
```

with that being said though, I was able to run both and it worked. Maybe its a problem with that missing semi-colon after readonly const?

Also; could you be more specific with what you mean by "it doesnt work"—what doesnt work?

**Comment by xavi** [comment]
By not working I meant that an element with both the `.checked` and `.readonly` classes was not displayed with a red background.

But... I tried adding the semicolons, and applying the `&` at the beginning and it worked. Then I went back to my original code (no semicolons, `&` at the end) and now it worked too!

It's weird. Maybe there was a weird invisible character that I deleted when adding the semicolons? That's the only explanation that I can think of for this behavior.
